scripts/copy-php-beta-models.ps1

```diff
@@ -0,0 +1,11 @@
+# removes all models under src/Beta
+Remove-Item $env:RepoModelsDir -Recurse -Force
```

Comment:
nit: could we refactor the v1 and beta script by passing a version argument?

Reply:
One removes existing files by excluding certain folders, the other removes by the folder name directly. One requires generation of a new folder, the other doesn't. It is almost like I will need a big if/else block to contain the contents of these two files. I think it is cleaner the way it is.